src/components/admin/lib/max-field-lengths.js

```diff
@@ -0,0 +1,5 @@
+/*
+ * File to store and export the maximum values for any text fields
+ */
+
+export const MAX_TEASER_LENGTH = 156;
```

Comment:
@emilgoldsmith This file is definitely overkill at the moment, so I'm not sure how I feel about this as it sort of is engineered with the expectation people will need at least a few more maximum field lengths. LMK what you think.

Reply:
You could also just call it constants.js for now, and we can break them out into several constants files later if we get a lot of different ones, yeah?

Reply:
Because it's more likely we have more constants of any kind